docs/quickstarts/insights-inventory-workspace-rbac/insights-inventory-workspace-rbac.yaml

```diff
@@ -1,10 +1,10 @@
 # Additional info: https://docs.openshift.com/container-platform/4.9/web_console/creating-quick-start-tutorials.html
 metadata:
-  name: insights-inventory-groups-rbac
+  name: insights-inventory-workspace-rbac
   # you can add additional metadata here
   instructional: true
 spec:
-  displayName: Managing user access with Inventory groups
+  displayName: Managing user access with Inventory workspaces
```

Comment:
Just realized this should have been capitalized before, and therefore now too. Should be "Workspaces" capital W